hbase-client/src/main/java/org/apache/hadoop/hbase/client/Admin.java

```diff
@@ -2276,6 +2276,18 @@ boolean snapshotCleanupSwitch(final boolean on, final boolean synchronous)
   List<SlowLogRecord> getSlowLogResponses(final Set<ServerName> serverNames,
       final SlowLogQueryFilter slowLogQueryFilter) throws IOException;
 
+  /**
+   * Retrieves online large RPC logs from the provided list of
+   * RegionServers
+   *
+   * @param serverNames Server names to get slowlog responses from
+   * @param largeLogQueryFilter filter to be used if provided
+   * @return online slowlog response list
+   * @throws IOException if a remote or network exception occurs
+   */
+  List<SlowLogRecord> getLargeLogResponses(final Set<ServerName> serverNames,
+    final SlowLogQueryFilter largeLogQueryFilter) throws IOException;
```

Comment:
@bharathv I am planning to change `SlowLogRecord` to `OnlineLogRecord` and `SlowLogQueryFilter` to `OnlineLogQueryFilter`. Will make that change before merging the PR because this class is being referred at multiple places (in thrift), so will be redundant refactor for now (to make review relevant to actual changes).

Also, the overall change is providing new API in Admin and shell command and to have 2 flags at server side (isSlowLog and isLargeLog) to fetch relevant records from server.